src/brevitas_examples/llm/main.py

```diff
@@ -285,6 +288,17 @@ def main(args):
 
     model = offload_model(model)
 
+    if args.learned_round:
+        print("Applying learned round...")
+
+        learned_round_llm_utils = LearnedRoundLLMUtils()
+        learned_round = AutoRound()
+        learned_round_optimiser = LearnedRoundOptimizer(
```

Comment:
Not a fan of this entrypoint, I'd rather have strings that have a meaning, that are interpreted within the class to the correct classes

Reply:
Agreed. I created a builder method. I don't think it's optimal still, and I'd be happy to iterate over it.